### DISCUSSION TITLE
How to actually replace magit-popup with transient?

### DISCUSSION THREAD

**Discussion by TLINDEN** [magit/magit#4910] [Support] [answered]
I am using this popup definition for years, it adds a `s` command to the log popup, which shows a dired buffer. It's easy to memorize, since I just type `ls` to reach it:
```
(magit-define-popup-action 'magit-log-popup
    ?s "Dired" 'magit-dired-jump)
```

How can I achieve the same effect with transient?

Thanks in advance,
Tom

**Comment by tarsius** [comment, marked as answer]
https://github.com/magit/magit/wiki/Converting-popup-modifications-to-transient-modifications#adding-an-action

**Comment by TLINDEN** [comment]
Thanks a lot! Oh I can't tell you how much I love it! :)
